recipes/cylowess/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cylowess" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b44cde6b7c3d4c7d35fa7ab93d1e1609ea966490183735f12c352d858c43c2ce
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - cython
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - cylowess
+
+about:
+  home: https://github.com/livingsocial/cylowess
+  license: UNKNOWN
```

Comment:
I opened an issue upstream, but I'm not confident this will be addressed, the library is 10 years old.

Reply:
I don't think we can distribute this in conda-forge if there is no license.

It seems to me that you need to track down the original author to clarify what license it can be published under.

Is this packaged in any other (Linux) distribution?